docs/choosing-a-site.rst

```diff
@@ -0,0 +1,43 @@
+Choosing between our two sites
+==============================
+
+A question our users often have is what the difference is between |org_brand| and |com_brand|.
+This page will lay out the functional and philosophical differences between the two sites,
+which should help you choose which is a better fit for your organization.
+
+The features available on both platforms are the same.
```

Comment:
This isn't quite true... There are more features in .com.

Reply:
Like what?

Reply:
Organizations, and sharing

Reply:
Yea, those are mentioned here. They aren't really functional differences.